spectrocloud/cluster_common.go

```diff
@@ -42,3 +44,17 @@ func toClusterResourceConfig(d *schema.ResourceData) *models.V1ClusterResourcesE
 		Rbacs:      toClusterRBACsInputEntities(d),
 	}
 }
+
+func toSSHKeys(cloudConfig map[string]interface{}) ([]string, error) {
+	var sshKeys []string
+	if cloudConfig["ssh_key"] != "" && len(cloudConfig["ssh_keys"].(*schema.Set).List()) == 0 {
```

Comment:
@SivaanandM could you please refactor using variables? 
`sshKey := cloudConfig["ssh_key"].(string)
sshKeysList := cloudConfig["ssh_keys"].(*schema.Set).List()` ?

Reply:
done